testsupport/has/api/v1alpha1/application_types.go

```diff
@@ -0,0 +1,91 @@
+/*
+Copyright 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ApplicationSpec defines the desired state of Application
```

Comment:
Let's mention that it's a dummy CRD we use only for tests, etc. I would also mention it in the test itself.
So no one get confused about e2e owning an Application API and relationship with the actual Application API from HAS.

Reply:
added comments in 3316925

Reply:
Good, descriptive comments. Thank you!